tools/pipelines/build-client.yml

```diff
@@ -129,7 +129,7 @@ extends:
     poolBuild: NewLarge
     checkoutSubmodules: true
     taskBundleAnalysis: true
-    taskLint: false
+    taskLint: true
```

Comment:
Does anyone know why this was disabled?

Reply:
Haven't gone through the logic myself to confirm, but this seems to be why: https://github.com/microsoft/FluidFramework/pull/15874#discussion_r1221860975

Reply:
Looks like it happened when we switched CI over to fluid-build. I suspect the `ci:build` script was intended to run `lint` as one of its steps, but it currently only runs `eslint` This seems like an oversight.

Reply:
I've confirmed that enabling all of `lint` instead of just `eslint` results in the right script being run, and everything passes as expected. I'll put up a separate PR for this.

Reply:
I think Curtis change was correct in that in maintained the behavior at the time; the script that ran due to `taskLintName: ci:eslint` was `"ci:eslint": "pnpm run -r --no-sort --stream eslint"`. But I agree that running the whole `lint` seems like a better option.

Reply:
#18566